LambdaEngine/Source/Game/Multiplayer/Client/ClientSystem.cpp

```diff
@@ -85,6 +87,26 @@ namespace LambdaEngine
 		return true;
 	}
 
+	void ClientSystem::SetServerHostID(uint32 serverHostID)
+	{
+		m_ServerHostID = (int32)serverHostID;
+	}
+
+	void ClientSystem::SetClientHostID(uint32 clientHostID)
+	{
+		m_ClientHostID = (int32)clientHostID;
+	}
+
+	int32 ClientSystem::GetServerHostID()
+	{
+		return m_ServerHostID;
+	}
+
+	int32 ClientSystem::GetClientHostID()
+	{
+		return m_ClientHostID;
+	}
+	
```

Comment:
There seem to be no member functions using the variables, except for the getter and setter. This should not be added to the Engine.